snmp/tests/test_unit.py

```diff
@@ -581,3 +581,61 @@ def test_batches(items, size, output):
 def test_batches_size_must_be_strictly_positive(size):
     with pytest.raises(ValueError):
         list(batches([1, 2, 3], size=size))
+
+
+def test_try_submit_bandwidth_usage_metric_if_bandwidth_metric():
```

Comment:
[nit] could add test case with `ifHighSpeed` == 0. 
Could be a goo use case for [property based testing](https://dev.to/meeshkan/from-1-to-10-000-test-cases-in-under-an-hour-a-beginner-s-guide-to-property-based-testing-1jf8)

Reply:
Addressed in https://github.com/DataDog/integrations-core/pull/8210/commits/19f6079b71c17a9a1832731ddf3b269363b984ed